pkg/kclient/generators.go

```diff
@@ -58,3 +56,20 @@ func GenerateDeploymentSpec(podTemplateSpec corev1.PodTemplateSpec) *appsv1.Depl
 
 	return deploymentSpec
 }
+
+// GeneratePVCSpec creates a pvc spec
+func GeneratePVCSpec(quantity resource.Quantity) *corev1.PersistentVolumeClaimSpec {
+
+	pvcSpec := &corev1.PersistentVolumeClaimSpec{
+		Resources: corev1.ResourceRequirements{
+			Requests: corev1.ResourceList{
+				corev1.ResourceStorage: quantity,
+			},
+		},
+		AccessModes: []corev1.PersistentVolumeAccessMode{
+			corev1.ReadWriteMany,
```

Comment:
We can't use ReadWriteMany volumes due to the lack of Kube storage drivers that support it.

Reply:
Updated to `RWO`